src/main/java/net/imagej/ops/features/haralick/DefaultDifferenceVariance.java

```diff
@@ -47,32 +47,37 @@
  * @author Christian Dietz (University of Konstanz)
  * @author Tim-Oliver Buchholz (University of Konstanz)
  *
- * Formula based on http://murphylab.web.cmu.edu/publications/boland/boland_node26.html.
+ * Formula based on: http://haralick.org/journals/TexturalFeatures.pdf
  */
 @Plugin(type = Ops.Haralick.DifferenceVariance.class, label = "Haralick: Difference Variance")
-public class DefaultDifferenceVariance<T extends RealType<T>> extends
-		AbstractHaralickFeature<T> implements Ops.Haralick.DifferenceVariance {
+public class DefaultDifferenceVariance<T extends RealType<T>> extends AbstractHaralickFeature<T>
+		implements Ops.Haralick.DifferenceVariance {
 
 	private UnaryFunctionOp<double[][], double[]> coocPXMinusYFunc;
-	
+
 	@Override
 	public void initialize() {
 		super.initialize();
 		coocPXMinusYFunc = Functions.unary(ops(), CoocPXMinusY.class, double[].class, double[][].class);
 	}
-	
+
 	@Override
 	public void compute(final IterableInterval<T> input, final DoubleType output) {
 		final double[][] matrix = getCooccurrenceMatrix(input);
-
 		final double[] pxminusy = coocPXMinusYFunc.calculate(matrix);
+
+		double m = 0.0;
+
+		for (int i = 0; i < numGreyLevels; i++) {
+			m += i * pxminusy[i];
+		}
+
 		double sum = 0.0d;
-		for (int k = 0; k < pxminusy.length; k++) {
-			sum += k * k * pxminusy[k];
+		for (int k = 0; k < numGreyLevels; k++) {
+			sum += Math.pow(k - m, 2) * pxminusy[k];
```

Comment:
FWIW, I benchmarked `Math.pow(k - m, 2)` vs `(k-m) * (k-m)` and there is no difference in performance.